test/datasets/datasets_test.py

```diff
@@ -1,19 +1,16 @@
+from torchaudio.datasets.cmuarctic import CMUARCTIC
```

Comment:
Please keep changes minimum and revert changes related to formatting. Formatting changes update 'git blame' and it will make it hard to reason about the changes later.